device/routes/routeaddresses.yaml

```diff
@@ -0,0 +1,67 @@
+components:
+  schemas:
+    V4RouteAddress:
+      description: >-
+        A container for IPv4 route addresses.
+      type: object
+      required: [address]
+      properties:
+        address:
+          description: >-
+            The starting address of the network.
+          type: string
+          format: ipv4
+        prefix:
+          description: >-
+            The IPv4 network prefix length to be applied to the address. 
+          type: integer
+          default: 24
+          minimum: 0
+          maximum: 32
+        count:
+          description: >-
+            The total number of addresses in the range.
+          type: integer
+          default: 1
+          minimum: 1
+        step:
```

Comment:
No maximum?

Reply:
step and count will have variable max based on prefix length and hope bounded natural integer max int.